common/test-framework/org.jboss.tools.common.reddeer/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,24 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: RedDeer API for Common UI Tests
+Bundle-SymbolicName: org.jboss.tools.common.reddeer
+Bundle-Version: 3.8.3.qualifier
+Bundle-Activator: org.jboss.tools.common.reddeer.Activator
+Require-Bundle: org.eclipse.core.runtime,
+ org.jboss.reddeer.go;bundle-version="[1.2.0,2.0.0)",
```

Comment:
Bear in mind that this will need to be changed when RedDeer 2.0 comes out. I assume this is intentional but just wanted to call it out in case the plan is to make reddeer.go compatible between 1.2 and 2.0.